mollie-payments-for-woocommerce/includes/mollie/wc/gateway/abstract.php

```diff
@@ -394,29 +395,30 @@ protected function paymentConfirmationAfterCoupleOfDays ()
     }
 
 	/**
-	 * @param int $order_id
+     * @param int $orderId
 	 *
-	 * @throws \Mollie\Api\Exceptions\ApiException
 	 * @return array
+	 *@throws \Mollie\Api\Exceptions\ApiException
 	 */
-	public function process_payment( $order_id ) {
-		$order = Mollie_WC_Plugin::getDataHelper()->getWcOrder( $order_id );
-
-		if ( ! $order ) {
-			Mollie_WC_Plugin::debug( $this->id . ': Could not process payment, order ' . $order_id . ' not found.' );
+    public function process_payment($orderId)
+    {
+        $order = Mollie_WC_Plugin::getDataHelper()->getWcOrder($orderId);
 
-			Mollie_WC_Plugin::addNotice( sprintf( __( 'Could not load order %s', 'mollie-payments-for-woocommerce' ), $order_id ), 'error' );
+        if ( ! $order ) {
+            Mollie_WC_Plugin::debug(
+                $this->id . ': Could not process payment, order ' . $orderId . ' not found.'
+            );
+            Mollie_WC_Plugin::addNotice(
+                sprintf(__('Could not load order %s', 'mollie-payments-for-woocommerce'), $orderId),
+                'error'
+            );
 
-			return array ( 'result' => 'failure' );
```

Comment:
@XedinUnknown This is because there's no coding standard, I would concentrate more to other stuffs for now. 
I would talk about coding standard when we'll have one.